service/dap/server.go

```diff
@@ -735,9 +710,15 @@ func cleanExeName(name string) string {
 }
 
 func (s *Server) onLaunchRequest(request *dap.LaunchRequest) {
+	var args LaunchConfig
+	if err := mapToStruct(request.Arguments, &args); err != nil {
```

Comment:
I think that the logicaly thing would be to change dap.LaunchRequest.Arguments to be a json.RawMessage.

Reply:
Proposed https://github.com/google/go-dap/pull/57

Reply:
once the pr is merged and go-dap is released, I will update this pr to use the new api.